Tests/iaas/scs-0104-v1-images.yaml

```diff
@@ -5,7 +5,7 @@ images:
   - https://cloud-images.ubuntu.com/jammy/
   status: mandatory
 - name: "ubuntu-capi-image"
-  name_scheme: "ubuntu-capi-image-v[0-9].[0-9]+(.[0-9]+)?"
+  name_scheme: "ubuntu-capi-image v[0-9].[0-9]+(.[0-9]+)?"
```

Comment:
FTR, the major version is restricted to single digit here but that should be fine for a loooong time so I don't think it needs to change just noticed ;)